test/test_replaygain.py

```diff
@@ -132,6 +141,20 @@ def test_cli_saves_album_gain_to_file(self):
 class ReplayGainGstCliTest(ReplayGainCliTestBase, unittest.TestCase):
     backend = u'gstreamer'
 
+    def setUp(self):
+        try:
+            # Check if required plugins can be loaded by instantiating a
+            # GStreamerBackend (via its .__init__).
+            config['replaygain']['targetlevel'] = 89
```

Comment:
This was needed as `GStreamerBackend.__init__` passes the config value to `_rg.set_property()` (which is present if run outside the test, as the plugin initialization takes care of creating the config value).